README.md

```diff
@@ -154,11 +186,41 @@ produce early errors because of recursive application of the first rule.
 - Let _targetName_ be ? Get(_target_, `"name"`).
 - If Type(_targetName_) is not `String`, then let _targetName_ be the empty string.
 - Let _status_ be ? SetFunctionName(_F_, _targetName_, `"bound"`).
+- Assert: _status_ is never used.
 - Return _F_.
 
 
 ----
 
+    BindExpression:
+        LeftHandSideExpression :: new
+
+- Let _targetReference_ be the result of evaluating _LeftHandSideExpression_.
+- Assert: IsSuperReference(_targetReference_) is `false` if Type(_targetReference_) is Reference.
+- Let _target_ be GetValue(_targetReference_).
+- If IsConstructor(_target_) is `false`, throw a `TypeError` exception.
+- Let _bound_ be a new bound constructor wrapper function as defined in Bound Constructor Wrapper Functions.
+- Set the [[BoundTargetConstructor]] internal slot of _bound_ to _target_.
+- Let _F_ be ? BoundFunctionCreate(_bound_, `undefined`, «»).
+- Return InitializeBoundFunctionProperties(_F_, _target_).
+
+
+----
+
+    BindExpression :
+        LeftHandSideExpression :: [lookahead ≠ new] MemberExpression
+
+- Let _baseReference_ be the result of evaluating _LeftHandSideExpression_.
+- Let _baseValue_ be GetValue(_baseReference_).
+- Let _targetReference_ be the result of evaluating _MemberExpression_.
+- Let _target_ be GetValue(_targetReference_).
+- If IsCallable(_target_) is `false`, throw a `TypeError` exception.
+- Let _F_ be ? BoundFunctionCreate(_target_, _baseValue_, «»).
```

Comment:
Remove this line ; )

I only have a second right now; I'll explain a bit more later.